### PR TITLE
myu's theme docu update v1

### PR DESCRIPTION
**Pull request by myumichi**
## Changes Made within the Documentation

- Specified more clearly that using Discord's CDN as a host no longer works, and that other hosts should be used. 

> **Section Descriptions**
- Updated section descriptions to be more clear of what the strings below change. 

> String Ordering
- Changed the string ordering to match what appears in Themer. 

### Specific String Changes
**Accent Strings**

| String          | Addition                 |
| ---------------- |:-----------------------:|
| brand up to 900 | **Removed,** information was useless. |
| brand_500_alpha_20 | **Added.** |
| brand_new_260 | **Added.** |
| brand_new_500 | **Added.** |
| brand_new_560 | **Updated,** made more concise. |

**Primary Strings**
| String          | Addition                 |
| ---------------- |:-----------------------:|
| primary_dark_200 | **Updated,** made more concise. |
| primary_dark_300 | **Updated,** made more concise, added extra information. |
| primary_dark_330 | **Updated,** removed outdated information. |
| primary_dark_360 | **Updated,** added extra information. |
| primary_dark_400 | **Updated,** removed outdated information. |
| primary_dark_630 | **Updated,** made more concise, added extra information. |
| primary_dark_660 | **Updated,** improved clarity. |

**General Strings**
| String          | Addition                 |
| ---------------- |:-----------------------:|
| black_alpha_10 | **Added.** |
| primary_300 | **Updated,** added extra information. |
| primary_630 | **Updated,** added extra information. |
| primary_700 | **Updated,** made more concise, removed outdated information. |
| primary_700_alpha_80 | **Updated,** made more concise. |
| black_alpha_80 | **Updated,** made more concise. |
| white | **Updated,** added missing information. |
| white_500 | **Updated,** made more concise, added extra information. |

**Drawables**
| String          | Addition                 |
| ---------------- |:-----------------------:|
| drawable_button_grey | **Updated,** improved clarity. |
| drawable_open_folder_dark | **Added.** |

### CODE REVIEW

File: theme-dev/README.md [hunk 1]
```diff
@@ -202,8 +206,7 @@ These are provided by the plugin (or Xposed module). They theme many things at o
 
 ## UIKit Strings
 
-> UIKit strings, this will be updated when I figure out more details relating to this.
-
+> UIKit strings are used for brand colors that aren't blurple, along with text and buttons.**
```
Comment (GangsterFox) [inline theme-dev/README.md:209]: You forgot the ** at the start of the text

**Review by GangsterFox — CHANGES_REQUESTED** [review]
In general looks good to me, a nice cleanup and clarification, but needs some things fixed

File: theme-dev/README.md [hunk 2]
```diff
@@ -142,56 +141,61 @@ These are provided by the plugin (or Xposed module). They theme many things at o
...
 | String          | Purpose                 |
 | ---------------- |:-----------------------:|
-| brand up to 900 | Old brand colors (this string isn't used anymore, so it's useless) |
 | brand_new up to 900 | New brand Colors |
 | brand_new | Accent color |
-| brand_new_360 | Cursor color, nitro text color in the settings, turned on switch, etc... |
 | brand_new 230 to 630 | Accent color for buttons, bot tags, and On/Off sliders. |
-| brand_new_560 | Changes the reaction clicked border |
+| brand_500_alpha_20 | Channel / User mention background |
+| brand_new_260 | Channel / User mention text
```
[the remark this answers is not in the document]
Comment (myumichi) [reply theme-dev/README.md:153]: i'll fix this and the other one

**Comment by myumichi** [comment]
updated.

**Pull request merged**